src/components/PlayerList/PlayerList.spec.js

```diff
@@ -0,0 +1,22 @@
+import { render, screen } from '@testing-library/react'
+
+import PlayerList from './PlayerList'
+
+const testData = [
+  { name: 'player 1', id: 1 },
+  { name: 'player 2', id: 2 },
+  { name: 'player 3', id: 3 },
+]
+
+describe('PlayerList', () => {
+  it('renders', () => {
+    render(<PlayerList data={testData} />)
+    const playerListItem = screen.getByText('player 1')
```

Comment:
When you change `p` to `li` you could use `getByRole` instead.

Reply:
Thanks, implemented 👍🏾